### ISSUE TITLE
Make Accessing Projects Easier

### ISSUE THREAD

**Issue by avyrie** [melisalarson/easy-do#28]
When you add a project, it would be helpful to indicate that you can access the project by clicking on it when it is added to your current projects. It's easy to figure out as it is, but would be helpful if it was more obvious.

**Comment by JimmySpann** [comment]
How would you recommend we indicate it?

**Comment by avyrie** [comment]
I'll get back to you on that lol I just try to think of stupid-proofing when giving feedback

**Comment by JimmySpann** [comment]
Good idea! There are tons of other isssues/bugs/missing features in easyDo. Please post all the ones you see. We don't mind getting blown up haha

**Comment by avyrie** [comment]
I gotchu. Proud of you guys for continuing to develop this!
